.github/workflows/generate_policies.yml

```diff
@@ -35,11 +35,11 @@ jobs:
         git config --global user.email "12745184+zapbot@users.noreply.github.com"
         git config --global user.name $GITHUB_USER
         BRANCH=scan-policies-updt
-        cd zap-extensions
-        git remote add upstream https://github.com/zaproxy/zap-extensions.git
         SRC_BASE="zaproxy/zap-extensions@"$(git log -1 --format=format:%h)
```

Comment:
This is being done outside the git repo now.

Reply:
Thanks, good catch